app/src/main/java/agency/five/tmdb/HomeScreen.kt

```diff
@@ -0,0 +1,169 @@
+package agency.five.tmdb
+
+import agency.five.tmdb.ui.theme.MovieItemViewState
+import agency.five.tmdb.ui.theme.Typography
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+
+@Composable
+fun HomeScreen(
+    modifier: Modifier = Modifier,
+    map1: Map<String, List<MovieItemViewState>>, // map of movies for the "What's popular" section with keys as tab names
+    map2: Map<String, List<MovieItemViewState>>, // map of movies for the "Free to watch" section with keys as tab names
+    map3: Map<String, List<MovieItemViewState>>, // map of movies for the "Trending" section with keys as tab names
+    navController: NavController
+) {
+
+    // list of tabs for the "What's popular" section
+    val t1 = listOf(
+        stringResource(id = R.string.streaming_tab),
+        stringResource(id = R.string.on_tv_tab),
+        stringResource(id = R.string.for_rent_tab),
+        stringResource(id = R.string.in_theaters_tab)
+    )
+
+    // list of tabs for the "Free to watch" section
+    val t2 = listOf(
+        stringResource(id = R.string.movies_tab),
+        stringResource(id = R.string.tv_tab)
+    )
+
+
+    // list of tabs for the "Trending" section
+    val t3 = listOf(
+        stringResource(id = R.string.today_tab),
+        stringResource(id = R.string.this_week_tab)
+    )
+
+
```

Comment:
Go easy on these empty lines. Please remove any empy line that is not needed. It makes the code really hard to read.

If you are using that to structure your code and divide different components, that might be a sign to extract the components into functions